config/database.php

```diff
@@ -63,6 +63,20 @@
             'engine' => 'InnoDB ROW_FORMAT=DYNAMIC',
         ] + $db_config,
 
+        'mysql_testing' => [
```

Comment:
why different db connection config for testing? We can use the same connection but connect to a database for testing instead.